### PR TITLE
update to 2.0

### PR DESCRIPTION
**Pull request by SimonGreenhill**
updated to 2.0. All looks ok, except for the following warning:

```
/Users/simon/projects/lexibank2/env/lib/python3.7/site-packages/csvw/metadata.py:704: UserWarning: Unspecified column "IPA" in table None
  'Unspecified column "{0}" in table {1}'.format(k, self.local_name))
```

Any ideas?

### CODE REVIEW

File: lexibank_tryonsolomon.py [hunk 1]
```diff
@@ -35,26 +34,33 @@ class Dataset(BaseDataset):
...
-            ds.add_languages(id_factory=lambda l: slug(l["Name"]))
-            for i, (lang, param, value) in enumerate(cursor.fetchall(), 1):
-                if value:
-                    ds.add_lexemes(
-                        Language_ID=slug(lang),
-                        Parameter_ID=slug(param),
-                        Value=self.lexemes.get(value, value),
-                        Source=[source.id],
-                    )
+        args.writer.add_sources(source)
```
Comment (LinguList) [inline lexibank_tryonsolomon.py]: it's enough two write `args.writer.add_sources()`, it automatically searches for `sources.bib` in your `raw` folder.

File: lexibank_tryonsolomon.py [hunk 2]
```diff
@@ -35,26 +34,33 @@ class Dataset(BaseDataset):
...
+        args.writer.add_concepts(id_factory=lambda c: slug(c.label))
```
Comment (LinguList) [inline lexibank_tryonsolomon.py]: I suggest:

```python 
concept_lookup = args.writer.add_concepts(id_factory=lambda c: c.id.split('-')[-1]+'_'+slug(c.english),
    lookup_factory="Name")
language_lookup = args.writer.add_languages(lookup_factory="Name")
```

File: lexibank_tryonsolomon.py [hunk 3]
```diff
@@ -35,26 +34,33 @@ class Dataset(BaseDataset):
...
+        args.writer.add_languages(id_factory=lambda l: slug(l["Name"]))
+        
+        for lang, param, value in cursor.fetchall():
+            if value:
+                args.writer.add_forms_from_value(
+                    Language_ID=slug(lang),
```
Comment (LinguList) [inline lexibank_tryonsolomon.py]:
```python
Language_ID=language_lookup[lang],
Parameter_ID=concept_lookup(lang)
```

**Review by LinguList — COMMENTED** [review]
One can discuss the details, but it will make sure the IDs are more in line with the ones we discussed.

File: lexibank_tryonsolomon.py (hunk 1 above)
Comment (SimonGreenhill) [reply lexibank_tryonsolomon.py]: thanks Mattis - made all the requested changes in the next commit..

**Pull request merged**